analysis/blacklist/blacklist.go

```diff
@@ -6,49 +6,68 @@ import (
 	"net/http"
 	"os"
 
-	bl "github.com/activecm/rita-bl"
-	blDB "github.com/activecm/rita-bl/database"
+	rita_bl "github.com/activecm/rita-bl"
```

Comment:
Go lint didn't flag this, but can we change these variables to use mixedCaps instead of underscores?

Reply:
fixed :)